markup_tag_evaluation/awesome_align_tokenize.py

```diff
@@ -1,23 +1,34 @@
 #!/usr/bin/env python3
 import argparse
 from io import TextIOWrapper
-from awesome_align.tokenization_bert import BertTokenizer
-from awesome_align.tokenization_utils import PreTrainedTokenizer
+from awesome_align.tokenization_bert import BertTokenizer  # type: ignore
+from awesome_align.tokenization_utils import PreTrainedTokenizer  # type: ignore
```

Comment:
Why adding the type ignore?

Reply:
Otherwise, I get the following error when running the type checks with `./scripts/test.sh`:
```
markup_tag_evaluation/awesome_align_tokenize.py:4: error: Skipping analyzing "awesome_align.tokenization_bert": module is installed, but missing library stubs or py.typed marker  [import-untyped]
markup_tag_evaluation/awesome_align_tokenize.py:4: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
```